doc/audio.md

```diff
@@ -1,8 +1,17 @@
 # Audio
 
-To play an audio is really simple!
+To play audio is really simple!
 
-Start by adding an entry in your `pubspec.yaml` for each file you want to use (and make sure that the files exist in the paths that you provide).
+First you have to add [flame_audio](https://github.com/flame-engine/flame_audio) to your dependency
+list in your `pubspec.yaml` file:
+
+```yaml
+dependencies:
+  flame_audio: 0.1.0-rc4
+```
+
+Then you should add entries in the `assets` section for each file that you want to use (and make
```

Comment:
After installing the flame_audio package you can add audio files in the `assets` section of your pubspec.yaml file. Make sure that the audio files exists in the paths that you provide.